lib/FileCard/styles.scss

```diff
@@ -0,0 +1,123 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+$field-card-width: 200px !default;
+$field-card-height: 180px !default;
+$field-card-action-button-size: 32px !default;
+$field-card-margin: $layout-spacing-base !default;
+
+$field-card-label-font-size: $typo-size-slight !default;
+$field-card-actions-padding: .5rem !default;
+
+$field-card-label-bg: rgba(0, 0, 0, .7) !default;
+$field-card-highlight-color: rgb(245, 218, 116);
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+.file-card {
+  width: $field-card-width;
+  height: $field-card-height;
+  margin-right: $field-card-margin;
+  position: relative;
+  display: inline-block;
+  overflow: hidden;
+
+  &:hover,
+  .file-card__action:focus {
+    @for $i from 1 through 4 {
+      .file-card__action:nth-child(#{$i}) {
+        transform: translateY(0);
+        transition-delay: #{$i/20}s;
+      }
+    }
+  }
+
+  &.has-comments {
```

Comment:
It would be worth discussing where we should use state classes and BEM modifiers. Every BEM modifier could be argued as a state class but I'd rather keep these state classes to a select generic few e.g;

- `is-active`
- `is-disabled`
- `has-loaded` 

Things like `has-comments` could be abstracted into something like `file-card--highlight` to say what the styling is doing rather than why we're applying the style.

Pros and cons to both but we need to align as to when we should use either. E.g. is there a difference between the reason why we have `has-comments` and `file-card--no--preview`?